### PR TITLE
Angular2 webdriver test updates

### PR DESCRIPTION
**Pull request by thompsonja**
@enisoc 

This supersedes PR #1988.  I'll work on potential Travis caching and/or Dockerizing in a later PR.

### CODE REVIEW

File: test/vtctld2_web_test.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
-"""A vtctld2 webdriver test."""
+"""A vtctld2 webdriver test that tests the different views of status page."""
```
Comment (michael-berlin) [inline test/vtctld2_web_test.py]: You can keep the old more generic description here.

Comment (thompsonja) [inline test/vtctld2_web_test.py]: Done.

**Comment by thompsonja** [comment]
Also went ahead and checked in the compiled angular2 app into web/vtctld2/app.

**Comment by michael-berlin** [comment]
Josh, can you please write a minimal script to copy the compiled Angular 2 app and also add a Makefile target for it?

This way it's clear how to update the app.

**Comment by thompsonja** [comment]
A script isn't really necessary, running make build_web should do the trick as long as bootstrap_web.sh was run.

**Comment by michael-berlin** [comment]
Josh, maybe some context here:

The idea is to decouple the checked in version of the app from the own you're currently developing and working one.

The later one should not be under version control and it should not show up as a changed file when you rebuild it.

Therefore, we'll need a script which copies the current state to a dedicated folder where the "release" version of the app lives.

Does that make sense?

**Comment by michael-berlin** [comment]
Anthony said that we already have this distinction with the `dist` and the `app` folder and that only `bootstrap_web.sh` copies a new version to the `app` folder?

Just for the record: Which step in `bootstrap_web.sh` re-generates the `app` version?

**Comment by thompsonja** [comment]
Ok so here is what I'm thinking the flow is for someone making changes to the web app:
* Run bootstrap.sh
* Source dev.env
* Run bootstrap_web.sh
* Run make build_web whenever you need to rebuild to test a code change. This generates the build in the web/vtctld2/dist/ folder.
* Test using local example, vtcombo, etc., passing in web/vtctld2/dist as web_dir2.
* When happy, run tools/generate_web_artifacts.sh, which builds artifacts at web/vtctld2/app/ and removes the unnecessary assets folder.
👍 ? 👎 ?

**Comment by michael-berlin** [comment]
I've added a thumbs up to your comment :D

LGTM after comment is addressed :)

**Pull request merged**